content/docs/learning-guide/tutorials/ai/kythera-city-level-overview.md

```diff
@@ -0,0 +1,219 @@
+
+![The Polygon Prism Shape component](/images/learning-guide/tutorials/ai/kythera-city-navmesh-bounds.png)
+
+`Navmesh.xml` is customized to be consistent with the `Robot.prefab` (`KytRobot`) entity that contains the
+PhysX character controller for accurate obstacle avoidance and slope transitioning.
+
+![Customizing the navmesh](/images/learning-guide/tutorials/ai/kythera-city-navmesh-customize.png)
+
+```xml
+<AgentHeight type="float">1.720000</AgentHeight>
+<AgentMaxClimb type="float">0.750000</AgentMaxClimb>
+<AgentMaxSlope type="float">60.000000</AgentMaxSlope>
+<AgentRadius type="float">0.600000</AgentRadius>
+```
+
+* `AgentHeight` = Capsule height
+* `AgentMaxClimb` = Step height
+* `AgentMaxSlope` = Maximum slope angle
+* `AgentRadius` = Capsule radius
+
+## 3D octree generation & customizability
+
+Like the `Navmesh Bounds` component, the `Octree` component comes with a `Polygon Prism Shape` component.
+
+![Octree component configuration](/images/learning-guide/tutorials/ai/kythera-city-octree-dialog.png)
+
+The `3DOctree` entity's `Octree component` is an configured for accuracy at the cost of some performance, using a small cell size to represent the complex geometry of the level.
+
+![Octree debug draw](/images/learning-guide/tutorials/ai/kythera-city-octree.png)
+
+*   To generate the octree use the **Generate Octree** icon on the Kythera AI Toolbar,
+    then make sure to save the octree using the **Save Octree** button<br>
+    ![Regenerate Octree toolbar icon](/images/learning-guide/tutorials/ai/kythera-city-octree-regenerate.png)
+*   To show debug draw for the Octree, enter the following commands in the O3DE console:
+    *   `kyt_drawmaster 1`
+    *   `kyt_drawnavoctree 2`
+
+
+![Enabling octree debug draw in the O3DE console](/images/learning-guide/tutorials/ai/kythera-city-octree-console.png)
+
+## Kythera AI Inspector tool (behavior trees and debug draw)
+
+The [Kythera AI Inspector](/docs/user-guide/gems/reference/kythera-ai/introduction-to-the-inspector) is a web-based tool that allows you to author behavior trees (BTs) and access advanced debug draw features. To launch the Kythera AI Inspector, click on the **Launch Inspector** icon in the Kythera Toolbar.
+
+By default, the Inspector will show the blackboard view, where you can debug existing blackboard values, and enable or disable the debug draw options.
+
+![The blackboard view in the Kythera AI Inspector](/images/learning-guide/tutorials/ai/kythera-city-inspector-bbview.png)
+
+To enable debug draw, expand Console Variables or drag it to the side to create a new tab.
+Set console variable `Draw Master` to 1 and collapse console variables.
+
+Expand `debug options`, here you can disable/enable various specific debug draw features.
+
+To view the behavior trees, click on BT Editor at the top of the page and select a behavior tree name to edit.
+Make sure the Open3D game or editor isn't playing or simulating while editing the behavior trees.
```

Comment:
```suggestion
Make sure that the game or **O3DE Editor** isn't playing or simulating while editing the behavior trees.
```
* Be sure to use "O3DE" on subsequent reference.
* "O3DE Editor" in bold on first reference to this tool, per the [terminology guidelines](https://docs.o3de.org/docs/contributing/to-docs/terminology/).